cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -1004,6 +1063,41 @@ impl_runtime_apis! {
 				}
 			}
 
+			impl BridgeMessagesConfig<bridge_to_bulletin_config::WithRococoBulletinMessagesInstance> for Runtime {
+				fn is_relayer_rewarded(_relayer: &Self::AccountId) -> bool {
+					// we do not pay any rewards in this bridge
```

Comment:
nit: `is_relayer_rewarded == true` => "we do not pay any rewards" is little bit kind of confusing, but probably we can live with that

Reply:
Well, we can't test that the relayer is rewarded if we don't pay any rewards :) I could change a comment, if you have some better version :)